src/components/Widgets/Widget.tsx

```diff
@@ -180,39 +179,34 @@ export function Widget({
         partnerName === ThemesMap.Memecoins && tokens ? { allow: tokens } : {},
     };
   }, [
-    allowChains,
-    allowedChainsByVariant,
-    fromAmount,
+    starterVariant,
+    partnerName,
     fromChain,
     fromToken,
+    toChain,
+    toToken,
+    fromAmount,
+    allowChains,
+    allowedChainsByVariant,
+    configTheme?.allowedBridges,
+    configTheme?.allowedExchanges,
     i18n.language,
     i18n.languages,
-    integratorStringByType,
+    widgetTheme.config.appearance,
+    widgetTheme.config.theme,
+    multisigWidget,
     isMultisigSigner,
     multisigSdkConfig,
-    multisigWidget,
-    setWalletSelectMenuState,
-    starterVariant,
-    theme.breakpoints,
-    theme.palette.accent1.main,
-    theme.palette.grey,
-    theme.palette.mode,
-    theme.palette.surface1.main,
-    theme.palette.surface2.main,
```

Comment:
@dennyscode why so many changes inside this file?

Reply:
For that, I just updated the dependency array and it cleaned up some.